input-output-hk/cardano-sl
# [R120-4] Filter non-ASCII names

## Description

This is a cherry-pick of https://github.com/input-output-hk/cardano-sl/pull/2834/commits/89d6d8a0eb2062d7b56978e596569866533b1f1c , needed as an _interim_ fix for https://iohk.myjetbrains.com/youtrack/issue/R120-4 

## Linked issue

https://iohk.myjetbrains.com/youtrack/issue/R120-4

## Type of change

- [~] 🛠 New feature (non-breaking change which adds functionality)
- [~] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [~] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [~] 🔨 New or improved tests for existing code

## Developer checklist


## Testing checklist


## QA Steps

1. Use the latest installers on Win
2. Try to create a wallet named "Àľãń_ʍʛŇīʛĥŏļäŝ_Śţäġĩńğ_ŵåľľęţ"
3. Make sure it can be retrieved back with `https://localhost:8090/api/v1/wallets`.

## Screenshots (if available)
